### PR TITLE
Machinist Remap

### PR DESCRIPTION
**Pull request by Eyeveri**
The current machinist bay is extremely tight and hard to maneuver in. It's shape is odd, and players generally do not like using it. 
So, with some creative insight from @NobleRow, I remapped the entire workshop to replace where RnD and Exploratory chemistry  were. 

I moved exploratory chemistry and RnD to where the old machinist shop was. Xenoarch's artifact storage and equipment room has been merged, and telescience is where the old artifact storage room was. 

New Workshop - We do lose some maintenance space, but it is made up for by the improved playability of an extremely popular role. 
I also put the water tank in maintenance because most players do this already.
StrongDMM-2024-03-23 14 49 57

Modified xeno storage - The storage area is entirely independent from the equipment area. Nothing really changes, aside from layout and location. I might revisit this and give dedicated lockers instead of the racks for the suits, however.
Modified Xenoequipment and storage

New RnD/Chemlab - I tried a different layout to have RnD be visible to the hall again, but it was very bad due to the 3x3 requirement for the machinery to optimize QOL. There is less seating in this area, but I added folding chairs to the science storage area incase they needed more chairs. This map change does, however, make it so science is more 'consolidated'. RnD is now directly across from the RD's office, and much closer to the meeting room and storage area.
ADJUSTEDSCI

New telescience location - Honestly I just moved it to where it is now because it is barely played, but would still benefit xeno-arch the most, I think. 
TelescienceNew

### CODE REVIEW

**Comment by DreamySkrell** [comment]
image
missing wire

**Comment by DreamySkrell** [comment]
image
.

**Comment by DreamySkrell** [comment]
image
this looks really bad and breaks perspective, like the railing is on top of the tables somehow

**Comment by DreamySkrell** [comment]
image
all of these items should be shifted up so they appear actually on the table

**Comment by DreamySkrell** [comment]
image
same here

**Comment by DreamySkrell** [comment]
image
and here

**Comment by DreamySkrell** [comment]
image
same thing about the railings here

**Comment by DreamySkrell** [comment]
image
light overlapping with an air alarm

**Comment by DreamySkrell** [comment]
image
windoor with wrong name
also wrong type (desk/eastright)

**Comment by DreamySkrell** [comment]
image
wrong windoor type also

**Comment by DreamySkrell** [comment]
image
inconsistent turf types here - should be `/turf/simulated/floor/tiled/dark/full` under windows, and the highlighted turf should be rubber carpet

**Comment by DreamySkrell** [comment]
image
do not change density or pixel shift vendors

File: maps/sccv_horizon/code/sccv_horizon_areas.dm [hunk 1]
```diff
@@ -173,6 +173,12 @@
 	area_blurb = "The scents of oil and machine lubricant fill the air in this workshop."
 	area_blurb_category = "robotics"
 
+/area/operations/lower/machinist/surgicalbay
+	name = "Machinist Surgical Bay"
+	icon_state = "machinist_workshop"
+	area_blurb = "The scent of sterilized equipment mixed with anxiety and concern fill the air."
```
Comment (DreamySkrell) [inline maps/sccv_horizon/code/sccv_horizon_areas.dm:179]: Do not railroad feelings into player characters like this.
Why would every machinist feel "anxiety and concern" every single round they enter this room, even if just for a moment?
Why would any IPC or robot feel "anxiety and concern" even if they're just coming to get their lightly damaged arm or leg fixed?

**Comment by DreamySkrell** [comment]
image
do not change desc and desc_extended of this door. Same thing as the area blurb

**Comment by DreamySkrell** [comment]
image
image
image
same thing about shifting items up so they appear actually on the table

**Comment by DreamySkrell** [comment]
image
overlapping guestpass terminal and air alarm

**Comment by DreamySkrell** [comment]
image
no fake objects and especially ladders

**Comment by DreamySkrell** [comment]
image
turf changed here by accident

**Comment by DreamySkrell** [comment]
image
should be normal tiled floor turf under this door

**Comment by DreamySkrell** [comment]
image
overlapping light and acid dispenser here

**Comment by DreamySkrell** [comment]
image
overlapping sign and light

**Comment by DreamySkrell** [comment]
image
windoors on desks should be on the office/lab/workshop side, not on the hallway side, so people can leave stuff there from the outside even if closed

**Comment by DreamySkrell** [comment]
image
these window shutters should start open

**Comment by DreamySkrell** [comment]
image
all of these are `/obj/machinery/door/blast/shutters` and manually changing the density, icon state, and opacity
use `/obj/machinery/door/blast/shutters/open` instead

**Comment by DreamySkrell** [comment]
image
overlapping sign and light

**Comment by DreamySkrell** [comment]
image
don't put lights on windows

**Comment by DreamySkrell** [comment]
image
remove this shutter from the door - scientists can't really leave/enter if it's closed
and most (all?) other places don't have shutters on doors

**Review by DreamySkrell — CHANGES_REQUESTED** [review]
as above

**Comment by Eyeveri** [comment]
> image remove this shutter from the door - scientists can't really leave/enter if it's closed and most (all?) other places don't have shutters on doors

It is a lock down shutter.

**Comment by DreamySkrell** [comment]
image
missing `/obj/effect/floor_decal/industrial/outline/emergency_closet`

File: maps/sccv_horizon/code/sccv_horizon_areas.dm (hunk 1 above)
Comment (Eyeveri) [reply maps/sccv_horizon/code/sccv_horizon_areas.dm:179]: Edited

**Comment by Eyeveri** [comment]
@DreamySkrell All changes, with the exception of the fake object, have been addressed. I put a regular ladder down for now, but Noble is spriting a custom sprite to replace it. The item is for flavor, and will remain for now.

**Comment by DreamySkrell** [comment]
image

As I said on discord:

> If it had a sprite and name that does not look like it's just a ladder, and it had a coded in type, it would be fine.

This is still a fake object, and has a ladder sprite, even if different from normal functional ladders.

**Comment by Eyeveri** [comment]
> image
> 
> As I said on discord:
> 
> > If it had a sprite and name that does not look like it's just a ladder, and it had a coded in type, it would be fine.
> 
> This is still a fake object, and has a ladder sprite, even if different from normal functional ladders.

I am adding in a thing that is it's ob obj. However the sprite was approved by Matt, so it will not be changing.

**Comment by Eyeveri** [comment]
!wip

**Comment by Eyeveri** [comment]
!review

**Comment by DreamySkrell** [comment]
image
wrong access on this door, 7 (science) while it should be 29 (mechanist)

**Comment by DreamySkrell** [comment]
image
buttons should be shifted so they appear visibly on the table and not floating off of it

image
for example like this

**Comment by DreamySkrell** [comment]
image
same thing here, the box being partially on the table and partially on the railing breaks perspective

**Comment by DreamySkrell** [comment]
image
missing grate here

**Comment by DreamySkrell** [comment]
image
missing corner floor decal

**Comment by DreamySkrell** [comment]
image
do not put grates on top of tiled floors (should only be on plating)

**Comment by DreamySkrell** [comment]
image
do not change icon_state or density or pixel shift of functional objects like railings (or vending machines or the like)
especially icon_state - same thing as about fake objects, if the icon of the actual railing changes some day, this would break, and it's generally just really bad practice

if a new type for short railings is needed, it should be defined in code
but it should work consistently with other railings still and not have density set to zero

**Comment by DreamySkrell** [comment]
image
image
I think something like this would be better

**Review by DreamySkrell — APPROVED** [review]

**Review by nauticall — CHANGES_REQUESTED** [review]
These flaps don't do anything. You should remove them and then put either a wall where it was, or move the fuel tank over to that spot.
image

Red: Better to put a box of bodybags here rather than a single one.
Yellow: Please just use the regular railings here. No need to put one on the bottom etiher as it's already up against a wall.
image

Area mismatch.
image

While we're here, might want to look into making this area a bit cozier. Lengthen the chair into a sofa and put the disposal bin on the other side, maybe? Just an idea.
image

This requests console is unset. Needs to be configured for the Machinist's office.
image

Minor thing: rubber carpets already come with their own outlines (once they're fixed, at least) - there's no need to put these extra splines in unless you want some particular flair. I don't think they look particularly good, personally, seeing as they're pure white in stark contrast to the black-and-blue theming of the room. 
image

Avoid using capital case like this for IDs. Stick to lowercase; you can use underscores if needed.
image

This door has no req access. Anyone can open the door and clamber in once the shutter's opened.
image


Overall, I like it! Just needs a few extra changes.

**Review by nauticall — APPROVED** [review]
yeehaw

**Review by DreamySkrell — APPROVED** [review]

**Comment by blackwolf5505** [comment]
I know this has been merged, but I don't like the precedent this sets in that the machinist being too cramped is a good excuse to jam two whole science jobs into the same area. Please stop shafting science for the sake of other departments, this would have made more sense to have been moved near cargo than to shrink RnD and chemistry into a closet while moving tscience to some godforsaken corner.

**Comment by Eyeveri** [comment]
Science is undergoing a remap from me right now. I did this to shift some things around, but frankly RnD doesn't need to be huge. The test range exists for testing purposes, and the Chem lab barely got any smaller.

There was no where to move the machinist shop to by cargo that fit well. 

Just wait a few days and you'll see science getting expanded significantly.

**Comment by Eyeveri** [comment]
> I know this has been merged, but I don't like the precedent this sets in that the machinist being too cramped is a good excuse to jam two whole science jobs into the same area. Please stop shafting science for the sake of other departments, this would have made more sense to have been moved near cargo than to shrink RnD and chemistry into a closet while moving tscience to some godforsaken corner.

Also this was up for 2 weeks before it was merged, no one had an issue with it.

**Comment by blackwolf5505** [comment]
RnD doesn't need to be huge in the same vein that the machinst shop doesn't need to be huge, but its the hub for the department and sees almost every science character in and out of it. It currently feels like a closet, and Tscience being downstairs ensures that whoever plays that gets even LESS interaction than they did before. Having it shunted down the hallway makes it feel even more isolated than it was previously.

My fear is that even with a remap, these issues aren't going to be fixed, and science will just become more and more decrepit as has been the case since the horizon launched and every little mapping change to it has been a net negative. We wonder why people don't play science when they're slowly relegated less and less space and are at the point that sitting in a closet smaller than the LOBBIES of other departments is acceptable. Forgive me if thinking that kind of 'temporary' change while you work on the bigger picture kills any faith I have of good intention.

If you want feedback for a PR you ought to be posting it on the fourms - not everyone is going to be checking github for these kinds of changes, particularly when its disingenuously written as a machinist remap when science was far more impacted by it. Beyond that, attempting to justify anything because a job is "popular" is insane. If anything, you'd want to give the neglected roles MORE love so they have a reason to exist.

**Comment by Eyeveri** [comment]
> If you want feedback for a PR you ought to be posting it on the fourms - not everyone is going to be checking github for these kinds of changes, particularly when its disingenuously written as a machinist remap when science...

I also posted _**several**_ updates to the mapper coordination channel. 

I addressed every single change in the PR when I made it, and, once again, it was up and available to be seen by the public for two weeks before implementation. 

Please do not insult me, thanks.

**Comment by blackwolf5505** [comment]
I'm not insulting you.

I'm giving you feedback, now, because only now did I see it. Trying to dismiss that because "well it was up for two weeks why didn't you see it" is amazingly frustrating. I told you why I think it's a strange choice, and I've told you how it negatively impacts people who play those roles for the sake of giving a well played role more love than it needs.

**Pull request merged**